src/test/scala/org/broadinstitute/dsde/firecloud/service/NihApiServiceSpec.scala

```diff
@@ -0,0 +1,93 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.dataaccess.{MockAgoraDAO, MockRawlsDAO, MockSearchDAO, MockThurloeDAO}
+import org.broadinstitute.dsde.firecloud.mock.MockGoogleServicesDAO
+import org.broadinstitute.dsde.firecloud.model.JWTWrapper
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.utils.DateUtils
+import spray.http.StatusCodes._
+
+import scala.concurrent.ExecutionContext
+
+class NihApiServiceSpec extends ApiServiceSpec {
+  val dbGapAuthorizedUsersGroupName = FireCloudConfig.Nih.rawlsGroupName
+  val validJwtOnWhitelist = JWTWrapper("eyJhbGciOiJIUzI1NiJ9.ZmlyZWNsb3VkLWRldg.NPXbSpTmAOUvJ1HX85TauAARnlMKfqBsPjumCC7zE7s")
```

Comment:
this jwt decodes to `firecloud-dev`, which is on the test whitelist in the mock google dao